### PR TITLE
#167836787 User should be able to login (database)

### PR DESCRIPTION
**Pull request by nignanthomas**
#### What does this PR do?
User should be able to login .

#### Description of Task to be completed?
Create signin controller to query the database and login the user.
Write tests for signin route.
Create users endpoints for admin to view all users.

#### How should this be manually tested?
Clone the repo to your machine and run:
* `npm install`.
* `npm run dev`.
*create a postgres database named `dbname`.
* Download Postman.
* In Postman, create a new POST request tab and enter the following URL: `http://localhost:3000/api/v2/auth/signin`.
* Enter the following user object as raw/json under `Body` Section: 
{
	"email": "newuser@gmail.com",
	"password": "password"
}
* You shoud have s similar response to the following:  
{
    "status": 201,
    "message": "User Logged in Successfully",
    "data": {
        "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXlGYXJlciIsInN1YiI6MTEsImlhdCI6MTU2NTYxMTc5NDU2OCwiZXhwIjoxNTY1Njk4MTk0NTY4fQ.pBg7q30isrfzjjs3eNzDYylPx15MifHTq2jgWF-V0P8"
    }
}

#### What are the relevant pivotal tracker stories?
[#167836787](https://www.pivotaltracker.com/story/show/167836787)

#### Screenshots (if appropriate)
login-postman

### CODE REVIEW

File: server/v2/models/dbQuery.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ if (NODE_ENV === 'test') {
 const pool = new Pool({
   connectionString,
 });
+console.log('connectionString1', connectionString);
```
Comment (ekumamait) [inline server/v2/models/dbQuery.js]: @nignanthomas do not push `development tools` with `production code` so always make sure to remove all console.log functions

**Review by ekumamait — COMMENTED** [review]
The latest commit does not have a `story ID` so am thinking this commit does not get merged since it does not `Deliver` or `Finish` the story

File: server/v2/models/dbQuery.js (hunk 1 above)
Comment (nignanthomas) [reply server/v2/models/dbQuery.js]: Yes, sorry for that.
I forgot to remove it after debugging my code.

**Comment by nignanthomas** [comment]
> The latest commit does not have a `story ID` so am thinking this commit does not get merged since it does not `Deliver` or `Finish` the story

I will correct that too.
Thanks

**Pull request merged**